src/App.js

```diff
@@ -1,23 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
```

Comment:
why did you removed css file

Reply:
Since I deleted the App.css file, I needed to remove the import statement from the App.css file